templates/_javascripts/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Would it be possible to just link to bootstrap on a CDN, so we don't have to keep a copy?